skins/laika/tests/unit/components/pages/donation_confirmation/DonationConfirmation.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+import Buefy from 'buefy';
+import DonationConfirmation from '@/components/pages/DonationConfirmation.vue';
+import { createStore } from '@/store/donation_store';
+
+const localVue = createLocalVue();
+localVue.use( Vuex );
+localVue.use( Buefy );
+
+function getDefaultConfirmationData(): any {
+	return {
+		addressType: 'person',
+		address: {
+			city: 'Berlin',
+			countryCode: 'DE',
+			email: 'test@wikimedia.de',
+			firstName: 'Tester',
+			fullName: 'Prof. Dr. Tester McTest',
+			lastName: 'McTest',
+			postalCode: '10963',
+			salutation: 'Herr',
+			streetAddress: 'Tempelhofer Ufer 23-24',
+		},
+		donation: {
+			accessToken: 'a839bc8045aba4c8b600bc0477dbbf10',
+			amount: 12.35,
+			bankTransferCode: 'XW-XLK-M3F-Z',
+			id: 1,
+			interval: 0,
+			optsIntoDonationReceipt: true,
+			optsIntoNewsletter: false,
+			paymentType: 'UEB',
+			updateToken: 'd387cebd6cc05efbd117545492cb0e99',
+		},
+	};
+}
+
+describe( 'DonationConfirmation', () => {
+	it( 'displays bank data for bank transaction payments', () => {
+		let confirmationData = getDefaultConfirmationData();
+		const wrapper = mount( DonationConfirmation, {
+			localVue,
+			propsData: {
+				confirmationData: confirmationData,
+			},
+			store: createStore(),
+			mocks: {
+				$t: ( key: string ) => key,
+			},
+		} );
+
+		expect( wrapper.find( '#bank-data' ).html() ).toContain( 'XW-XLK-M3F-Z' );
```

Comment:
it's better to check for `confirmationData.donation.bankTransferCode` here instead of the hardcoded value.
This way we can't make a "human" mistake like a typo.